scripts/copy-build-to-dist.mjs

```diff
@@ -7,16 +7,13 @@ const PACKAGES_PATH = path.join(ROOT_DIR, "packages");
 const DEFAULT_BUILD_PATH = path.join(ROOT_DIR, "build");
 
 let activeOutputDir = DEFAULT_BUILD_PATH;
-if (process.env.REMIX_LOCAL_DEV_OUTPUT_DIRECTORY) {
-  let appDir = path.join(
-    ROOT_DIR,
-    process.env.REMIX_LOCAL_DEV_OUTPUT_DIRECTORY
-  );
+if (process.env.REMIX_LOCAL_BUILD_DIRECTORY) {
+  let appDir = path.join(ROOT_DIR, process.env.REMIX_LOCAL_BUILD_DIRECTORY);
```

Comment:
I don't think `copyBuildToDist` plays quite right with this - it seems to copy any `node_modules/*/dist` directory into `packages/*/dist` - not just remix packages.  Might just need something like the following in `getPackageBuildPaths`:

```js
      if (path.basename(moduleDir) === "@remix-run") {
        packageBuilds.push(...(await getPackageBuildPaths(moduleDir)));
      } else if (
        /node_modules\/@remix-run\//.test(moduleDir) ||
        /node_modules\/create-remix/.test(moduleDir) ||
        /node_modules\/remix/.test(moduleDir)
      ) {
        packageBuilds.push(moduleDir);
      }
```


I also think I noticed that `node_modules/remix` wasn't getting created inside of `REMIX_LOCAL_BUILD_DIRECTORY`.  That might only be an issue if you're actively changing `magicExports` stuff though so I don't think that's a big issue if it's a pain to handle